This will default the setting for concurrent_layout

teryhill opened the pull request:
This will default the concurrent_layout setting while that code is being removed. We should be able to remove it as we come to it in a controlled manner. #288 

$GLOBALS['concurrent_layout'] = 3;

teryhill commented:
I will do that now. the commit is on the way

tmccormi commented:
Looks good.  I assume it's testing fine as well?

teryhill commented:
I am running it locally with no issues. I removed the entry from the globals table and it continues to run. Pryor to submitting I changed the setting via globals and it did not change the left nav meu.

tmccormi commented:
I merged your changes into my WIP branch: concurrent_layout_cleanup  and
pushed it back up stream for you amusement

Tony McCormick, CTO
www.mi-squared.com
Support: 866-735-0897, Direct: 713-574-6709
My Calendar:  http://bit.ly/XznvDo

tóg go bog ē - Best advice from a pub owner in Ireland, to take it easy.



On Tue, Nov 29, 2016 at 4:16 PM, Terry Hill <notifications@github.com>
wrote:

> I am running it locally with no issues. I removed the entry from the
> globals table and it continues to run. Pryor to submitting I changed the
> setting via globals and it did not change the left nav meu.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/LibreHealthIO/LibreEHR/pull/289#issuecomment-263717614>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AARci2Xhx102rWUiEA2l9OPY7JJ-SB0jks5rDKQ5gaJpZM4K_XzA>
> .
>

-- 
Please be aware that e-mail communication can be intercepted in 
transmission or misdirected. Please consider communicating any sensitive 
information by telephone.  The information contained in this message may be 
privileged and confidential. If you are NOT the intended recipient, please 
notify the sender immediately with a copy to hipaa-security@mrsb-ltd.com and 
destroy this message.

teryhill commented:
Where? 

      From: Tony McCormick <notifications@github.com>
 To: LibreHealthIO/LibreEHR <LibreEHR@noreply.github.com> 
Cc: Terry Hill <teryhill@yahoo.com>; Author <author@noreply.github.com>
 Sent: Tuesday, November 29, 2016 6:50 PM
 Subject: Re: [LibreHealthIO/LibreEHR] This will default the setting for concurrent_layout (#289)
   
I merged your changes into my WIP branch: concurrent_layout_cleanup and
pushed it back up stream for you amusement

Tony McCormick, CTO
www.mi-squared.com
Support: 866-735-0897, Direct: 713-574-6709
My Calendar: http://bit.ly/XznvDo

tóg go bog ē - Best advice from a pub owner in Ireland, to take it easy.



On Tue, Nov 29, 2016 at 4:16 PM, Terry Hill <notifications@github.com>
wrote:

> I am running it locally with no issues. I removed the entry from the
> globals table and it continues to run. Pryor to submitting I changed the
> setting via globals and it did not change the left nav meu.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/LibreHealthIO/LibreEHR/pull/289#issuecomment-263717614>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AARci2Xhx102rWUiEA2l9OPY7JJ-SB0jks5rDKQ5gaJpZM4K_XzA>
> .
>

-- 
Please be aware that e-mail communication can be intercepted in 
transmission or misdirected. Please consider communicating any sensitive 
information by telephone. The information contained in this message may be 
privileged and confidential. If you are NOT the intended recipient, please 
notify the sender immediately with a copy to hipaa-security@mrsb-ltd.com and 
destroy this message.
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.

tmccormi commented:
I just pushed a branch...  I shared that link earlier.  Here it is again.
https://github.com/LibreHealthIO/LibreEHR/tree/concurrent_layout_cleanup


Tony McCormick, CTO
www.mi-squared.com
Support: 866-735-0897, Direct: 713-574-6709
My Calendar:  http://bit.ly/XznvDo

tóg go bog ē - Best advice from a pub owner in Ireland, to take it easy.



On Tue, Nov 29, 2016 at 7:04 PM, Terry Hill <notifications@github.com>
wrote:

> Where?
>
> From: Tony McCormick <notifications@github.com>
> To: LibreHealthIO/LibreEHR <LibreEHR@noreply.github.com>
> Cc: Terry Hill <teryhill@yahoo.com>; Author <author@noreply.github.com>
> Sent: Tuesday, November 29, 2016 6:50 PM
> Subject: Re: [LibreHealthIO/LibreEHR] This will default the setting for
> concurrent_layout (#289)
>
> I merged your changes into my WIP branch: concurrent_layout_cleanup and
> pushed it back up stream for you amusement
>
> Tony McCormick, CTO
> www.mi-squared.com
> Support: 866-735-0897, Direct: 713-574-6709
> My Calendar: http://bit.ly/XznvDo
>
> tóg go bog ē - Best advice from a pub owner in Ireland, to take it easy.
>
>
>
> On Tue, Nov 29, 2016 at 4:16 PM, Terry Hill <notifications@github.com>
> wrote:
>
> > I am running it locally with no issues. I removed the entry from the
> > globals table and it continues to run. Pryor to submitting I changed the
> > setting via globals and it did not change the left nav meu.
> >
> > —
> > You are receiving this because you commented.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/LibreHealthIO/LibreEHR/pull/
> 289#issuecomment-263717614>,
> > or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/
> AARci2Xhx102rWUiEA2l9OPY7JJ-SB0jks5rDKQ5gaJpZM4K_XzA>
> > .
> >
>
> --
> Please be aware that e-mail communication can be intercepted in
> transmission or misdirected. Please consider communicating any sensitive
> information by telephone. The information contained in this message may be
> privileged and confidential. If you are NOT the intended recipient, please
> notify the sender immediately with a copy to hipaa-security@mrsb-ltd.com
> and
> destroy this message.
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/LibreHealthIO/LibreEHR/pull/289#issuecomment-263751384>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AARciwQFJleKsbAQYclUfDy76xXr-IaAks5rDMuRgaJpZM4K_XzA>
> .
>

-- 
Please be aware that e-mail communication can be intercepted in 
transmission or misdirected. Please consider communicating any sensitive 
information by telephone.  The information contained in this message may be 
privileged and confidential. If you are NOT the intended recipient, please 
notify the sender immediately with a copy to hipaa-security@mrsb-ltd.com and 
destroy this message.
